.travis.yml

```diff
@@ -34,6 +34,7 @@ script:
   - bin/test
   - bin/client_build
   - bin/server_build
+  - if [[ $TRAVIS_BRANCH == 'master' ]]; then bin/integration_test.sh; fi
```

Comment:
add double quotes for travis branch to avoid bash problems on eval.
Like this "$TRAVIS_BRANCH"

Reply:
Done.